spec/arachnid2_spec.rb

```diff
@@ -103,7 +177,7 @@
       allow(File).to receive(:open).with(Arachnid2::MEMORY_USE_FILE, 'rb').and_return(use_file)
       allow(File).to receive(:open).with(Arachnid2::MEMORY_LIMIT_FILE, 'rb').and_return(limit_file)
 
-      expect(@spider.send(:memory_danger?)).to be_truthy
+      expect(dummy.send(:memory_danger?)).to be_truthy
```

Comment:
why do you need 'send' here, 'memory_danger?' is a public method of Exoskeleton module?